packages/components/src/components/primitives/TextPairing/TextPairing.tsx

```diff
@@ -5,19 +5,19 @@ import { TextPairingProps, TextPairingStyles } from '.';
 import { useStyleConfig } from '@chakra-ui/system';
 import { useResponsiveValue } from 'hooks/useResponsiveValue';
 
-export const TextPairing: FC<TextPairingProps> = ({
-  variant,
-  label,
-  subLabel,
-}) => {
+export const TextPairing: FC<TextPairingProps> = ({ variant, label, subLabel }) => {
   const responsiveVariant = useResponsiveValue(variant);
   const styles = useStyleConfig('TextPairing', {
     variant: responsiveVariant,
   }) as TextPairingStyles;
   return (
     <Stack spacing={styles.columnGap}>
-      <Text variant={styles.label.variant}>{label}</Text>
-      <Text variant={styles.subLabel.variant}>{subLabel}</Text>
+      <Text variant={styles.label.variant} color={styles.label.color}>
+        {label}
+      </Text>
+      <Text variant={styles.subLabel.variant} color={styles.subLabel.color}>
+        {subLabel}
+      </Text>
```

Comment:
Probably we should accept two new properties in `TextPairing` -> `labelColor` and `subLabelColor` instead of creating this new variant

Reply:
Done!